### ISSUE TITLE
The card does not look like it is shown in README

### ISSUE THREAD

**Issue by bessarabov** [SNoof85/lovelace-tempometer-gauge-card#8]
I've using card from commit c1e0c4c5725360114512de5ba39ad1b754582ed8

Here is the config I'm using:

```yaml
type: 'custom:tempometer-gauge-card'
entity: sensor.0x00158d0004605ef0_temperature
min: 15
max: 30
```

This is how it looks:
![Screenshot 2020-08-17 at 17 00 02](https://user-images.githubusercontent.com/47263/90404806-8c886580-e0ab-11ea-814b-142faa6275e9.png)

This does not look the way I expected.

I checked the version from tag 0.1, but it looks the same.

I've checked this card in two browsers on my macbook: google chrome  84.0.4147.125 (WebKit 537.36) & Safari 13.1 (WebKit 605.1.15)

**Comment by SNoof85** [comment]
Ah ok. I'll try to reproduce it tonight then. There was some theme changes in last couples releases.

**Comment by SNoof85** [comment]
Fixed with 0.3 release. Do not use 0.2 wich has also a small issue.

**Comment by bessarabov** [comment]
Yes, everything is fine now, thank you!
